docs/apis.md

```diff
@@ -335,7 +335,11 @@ const customersApi = api('customers');
 
 const getAllCustomers = (ctx) => {};
 
-customersApi.get('/customers', validate, getAllCustomers);
+// Inline using .get()
+customersApi.get('/customers', [validate, getAllCustomers]);
+
+// Using .route()
+customersApi.route('/customers').get(validate, getAllCustomers);
```

Comment:
This doesn't seem correct to me. The first param for get is MethodOptions. Then middleware.